stdlib/public/stubs/Unicode/Common/GraphemeData.h

```diff
@@ -18,87 +18,86 @@
 
 #include "../SwiftShims/SwiftStdint.h"
 
-#define GRAPHEME_BREAK_DATA_COUNT 621
+#define GRAPHEME_BREAK_DATA_COUNT 610
 
-static __swift_uint32_t _swift_stdlib_graphemeBreakProperties[621] = {
+static const __swift_uint32_t _swift_stdlib_graphemeBreakProperties[610] = {
```

Comment:
This is the only change I wanted to make (adding the `const` keyword) but the gen-unicode-data package seems outdated — I had to rename input/output paths, etc. (And the generated data is different.)

Reply:
You can't say `GRAPHEME_BREAK_DATA_COUNT` there, I take it, instead of repeating the magic number?

Reply:
> You can't say `GRAPHEME_BREAK_DATA_COUNT` there, I take it, instead of repeating the magic number?

If the array is declared with a size, but its initializer list contains *fewer* elements, the rest will be *zero* initialized.

So the safest way might be to declare it with empty `[]` brackets, and then `static_assert` with the expected size and the `sizeof` operator.

(But this is probably unnecessary for auto-generated declarations.)

Reply:
Oh it seems I forgot to push my changes to this target. If you want, I can push those so that this is just the const addition?

Reply:
> If you want, I can push those so that this is just the const addition?

It might be easier if you add the `const` keyword yourself, when pushing those changes. If so, this PR can be closed.